src/Symfony/Component/Validator/Constraints/Valid.php

```diff
@@ -43,4 +43,11 @@ public function addImplicitGroupName(string $group)
             parent::addImplicitGroupName($group);
         }
     }
+
+    public function __construct(array $options = null, bool $traverse = null, array $groups = null, $payload = null)
```

Comment:
shouldn't $traverse be the last argument to not break BC when creating an instance?

Reply:
I did it similarly to the constraints that were fixed in https://github.com/symfony/symfony/commit/fb99eb2052523cfa2a9ba36006d020796548aaa6, in those argument order wasn't honored either. Why would that be an issue for this constraint?

Reply:
/cc @derrabus I may miss something here

Reply:
I think it does not hurt to not break BC anyway, right? Let's reorder.
Also, the `__construct` method should be moved as the first method of the class.

Reply:
@fabpot reordered, thanks for the remark. I actually just ran into an issue where the argument order was not honored (the `Callback` constraint) so I think keeping the BC is the right decision 👍🏼